blade-graphics/src/gles/command.rs

```diff
@@ -737,6 +775,30 @@ impl super::Command {
             Self::SetDrawColorBuffers(count) => {
                 gl.draw_buffers(&COLOR_ATTACHMENTS[..count as usize]);
             }
+            Self::SetBlendState(i, blend) => {
+                if let Some(blend_state) = blend {
+                    gl.enable(glow::BLEND);
+                    gl.blend_func_separate_draw_buffer(
+                        i,
+                        blend_state.color.src_factor.to_gles(),
+                        blend_state.color.dst_factor.to_gles(),
+                        blend_state.alpha.src_factor.to_gles(),
+                        blend_state.alpha.dst_factor.to_gles(),
+                    );
+                    gl.blend_equation_draw_buffer(i, blend_state.color.operation.to_gles());
+                } else {
+                    gl.disable(glow::BLEND);
+                }
+            }
+            Self::SetWriteMask(i, write_mask) => {
```

Comment:
There is another problem here - [glColorMaski](https://registry.khronos.org/OpenGL-Refpages/es3/html/glColorMask.xhtml) is only supported on GLES-3.2 (or with "EXT_draw_buffers_indexed").
The GLES backend in Blade should work on WebGL2, which is almost plain GLES-3.0, so if we just merge this all WebGL2 stuff will be broken immediately.

Here is what we could do. At pipeline creation, we'd detect if the color masks are the same for all buffers, or different. Same for the Blend functions. If it's not the same, and we don't support it (i.e. we are in GLES-3.1 or below, and we don't have this extension), we could at least panic and tell the user that GLES can't do this pipeline.
Then, when it's time to set the state, we'd have a condition somewhere that makes it set the state on all of the draw buffers at once.